sys/contrib/openzfs/contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -449,7 +449,7 @@ static int
 zfs_key_config_load(pam_handle_t *pamh, zfs_key_config_t *config,
     int argc, const char **argv)
 {
-	config->homes_prefix = strdup("rpool/home");
+	config->homes_prefix = strdup("zroot/home");
```

Comment:
Though it's ok, it would be better to have a FreeBSD ifdef and bring in the change via upstream.

Reply:
Should I just take this directly (with ifdefs) to OpenZFS, then? (Am I reading that right?) It doesn't seem super robust on FreeBSD to begin with; I can get it to mostly work with console logins, but SSH appears to be a non-starter (session complains about being unable to get password from PAM stack.)

Reply:
My apologies for the tardy response.... I've been away due to some personal issues...

I think that you likely should... But I'm not familiar with all the bits that are needed to make this functionality work completely on FreeBSD.

I've not tried this feature out at all...

Reply:
For upstream should we have a more extensive change to use `zroot` rather than `rpool` generally on FreeBSD?